datemath/helpers.py

```diff
@@ -101,15 +106,16 @@ def parse(expression: str, now: Any = None, tz: str = 'UTC', type: Any = None, r
         if debug: print("parse() - will now convert tz to {0}".format(tz))
         now = now.to(tz)
 
+    expression = str(expression)
```

Comment:
Less runtime casting in some cases. But more importantly, this is now type-safe.